pages/common/git-fork.md

```diff
@@ -0,0 +1,13 @@
+# git fork
+
+> Fork the given github repo. Like clone but forks first.
+> Part of git-extras.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-fork>.
+
+- Fork a given github repo with full url:
+
+`git fork {{https://github.com/tldr-pages/tldr}}`
+
+- Fork a given github with short name:
```

Comment:
I believe these are commonly called "slugs":
```suggestion
- Fork the specified GitHub repo by slug:
```

Reply:
Who knows that! 
No, I don't like this one

Reply:
I've never heard of `slugs` and even though the command itself is self-expantory, I'm not sure if `short name` would be easier to understand.

Reply:
Sure, let's change that.

Reply:
I actually wanted to comment on this since I knew there was a real name for it, but I just couldn't remember it. 
I think "slug" would be better myself. 

> A repository slug is a URL-friendly version of a repository name, automatically generated by Bitbucket for use in the URL. For example, if your repository name was 'føøbar', in the URL it would become 'foobar'. Similarly, 'foo bar' would become 'foo-bar'.     
> \- https://support.atlassian.com/bitbucket-cloud/docs/what-is-a-slug/

> Who knows that!

Maybe this is something people should know! \\(^-^)/

Reply:
TIL -- I had no idea this was a `slug`... I think we all learn everyday and it makes me happy  :)

Reply:
Yeah, people might be curious about the word and then look it up 💡

Reply:
Yes ... :)